web/src/scheduled_messages.js

```diff
@@ -59,44 +60,31 @@ export function is_send_later_timestamp_missing_or_expired(
     return false;
 }
 
-function sort_scheduled_messages_data() {
-    scheduled_messages_data.sort(
-        (msg1, msg2) => msg1.scheduled_delivery_timestamp - msg2.scheduled_delivery_timestamp,
-    );
-}
-
 function hide_scheduled_message_success_compose_banner(scheduled_message_id) {
     $(
         `.message_scheduled_success_compose_banner[data-scheduled-message-id=${scheduled_message_id}]`,
     ).hide();
 }
 
 export function add_scheduled_messages(scheduled_messages) {
-    scheduled_messages_data.push(...scheduled_messages);
-    sort_scheduled_messages_data();
+    for (const message of scheduled_messages) {
```

Comment:
Please call this `const scheduled_message`; we should reserve `const message` for objects that are messages.